components/eam/src/physics/cam/cloud_diagnostics.F90

```diff
@@ -53,12 +53,13 @@ subroutine cloud_diagnostics_register
     use phys_control,  only: phys_getopts
     use physics_buffer,only: pbuf_add_field, dtype_r8, dtype_i4
 
-    character(len=16) :: rad_pkg, microp_pgk
+    character(len=16) :: rad_pkg, microp_pgk, macrop_pgk
```

Comment:
I think you mean pkg instead of pgk?

Reply:
I just copied 'microp_pgk' which was already there.

Reply:
Yeah, but micro_pgk was obviously a typo!